src/test/java/io/confluent/connect/elasticsearch/integration/README.md

```diff
@@ -0,0 +1,12 @@
+The Integration tests are built to run with `maven-surefire-plugin` in conjuction with 
+`docker-maven-plugin` by `io.fabric8` to set up docker containers for Elasticsearch 
+(embedded nodes are no longer supported or recommended by Elasticsearch).
+
+To run the tests in their entirety, use `mvn verify`.
+
+To run the test manually (e.g. in Intellij IDEA),
+* Set up container with `mvn pre-integration-test`.
+* Run the tests manually
+* Tear down container with `mvn post-integration-test`.
+
+See the plugin specification in `pom.xml` for more details.
```

Comment:
do you want to describe how you created the certs here? would be good to keep the knowledge written down somewhere in case we want to regenerate them, or create new ones.

Reply:
My thinking is that this README would be for integration tests in general. I have made it easy to recreate the certs with the bash script `generate_certificates.sh` -- if you think it helps with clarity then I'll link it from the HTTPS integration test docs! If you feel strongly, then I can also reference it here

Reply:
Added in d597892bad91d91f69101c02247abc25d8ad5e95